### ISSUE TITLE
intestinal malrotation

### ISSUE THREAD

**Issue by sbello** [mgijax/mammalian-phenotype-ontology#3786]
consider adding term for:
intestinal malrotation
see HP:0002566
phenotype for Cornelia de Lange syndrome
annotations to abnormal intestine morphology, abnormal cecum position, abnormal duodenum topology,  mention this phenotype

need to consider how this relates to abnormal intestine placement 
def: different location or arrangement of the digestive tube passing from the stomach to the anus 
arrangement in this seems to cover rotation

also 
abnormal cecum position def: abnormal placement of the large sac normally located at the ileum and large intestine junction
abnormal duodenum topology def: abnormal position of the first division of the small intestine that extends from the pyloris to the junction with the jejunum 
neither of these sit below 'abnormal intestine placement'

would be good to review and improve the consistency of these terms

**Comment by sbello** [comment]
https://kidshealth.org/en/parents/malrotation.html 
"An intestinal malrotation is an abnormality that can happen early in pregnancy when a baby's intestines don't form into a coil in the abdomen" ... "Near the end of the first trimester of pregnancy, the intestines move from the umbilical cord into the abdomen. If they don't properly turn after moving into the abdomen, malrotation occurs. "

https://www.cincinnatichildrens.org/health/i/intestinal-malrotation
"Intestinal malrotation is a defect that occurs at the 10th week of gestation. During this stage of development, the intestines normally migrate back into the abdominal cavity following a brief period where they are temporarily located at the base of the umbilical cord. As the intestine returns to the abdomen, it makes two rotations and becomes fixed into its normal position"

Seems logical to make this a child of abnormal intestinal placement since failure to properly rotate will result in an abnormal arrangement of the intestines.

Added term for abnormal intestinal rotation MP:0014279

Need to check with HPO I used UBERON 'intestine' but HPO is using 'alimentary part of gastrointestinal system' not sure why they went up that high as this would include the stomach.
